### ISSUE TITLE
Align alttext requirement of EPUB Accessibility with that of WCAG 2.0/2.1

### ISSUE THREAD

**Issue by avneeshsingh** [w3c/epub-specs#1441]
In EPUB Accessibility 1.0 we had relaxed the referred WCAG requirement, success criteria 1.1.1 for text descriptions. At that time publishing industry had just started work on accessibility, and providing extended descriptions was a steep slope to climb at once, therefore we settled on alttext as a sufficient technique for conforming to EPUB Accessibility 1.0.

But over last 4 years publishing industry has made good progress for making publications accessible. Therefore we should consider making EPUB Accessibility 1.1 aligned with WCAG success criteria 1.1.1.

The relevant text from EPUB accessibility techniques is:

DESC-001: Include alternative text descriptions

[[WCAG20]] Success Criterion 1.1.1 requires a text alternative for all non-text content. In the case of image content,the provision of alternative text (e.g., in an [[HTML]] alt attribute) issufficient for meeting the requirements of the [[EPUB-A11Y]] specification.

Authors are strongly encouraged to include extended descriptions, but the utility of extendeddescriptions often depends on the needs of the user. Until the publishing ecosystem matures,making it easier for Authors to include extended descriptions and write them to optimally meetthe needs of the greatest number of users, they are not required.

If extended descriptions are not included, note the omission in the accessibility summary.

The requirement for a text alternative does not apply to decorative images that contribute noinformation to the text. These are identified by an empty alt attribute.

This guidance applies to the evaluation of EPUB Publications strictly for achievingconformance to the [[EPUB-A11Y]] specification. If an EPUB Publication is distributed in anenvironment where extended descriptions are required, this guidance for conformance will notbe sufficient.

The use of alternative text as a sufficient description for all images may be removed in afuture version of these techniques.

1. EPUB accessibility Techniques:
https://w3c.github.io/epub-specs/epub33/a11y-tech/index.html#desc-001

2. WCAG 2.1, SC 1.1.1
https://www.w3.org/TR/WCAG21/#non-text-content

**Comment by clapierre** [comment]
As part of Benetech's GCA program all publishers and conversion vendors (10 publishers & 5 vendors) who are participating all have been including extended descriptions on complex images.  I think we can remove this relaxed requirement and be truly aligned with 1.1.1

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#section2) on 2020-12-10)

resolution: Align text equivalent requirement of EPUB Accessibility with WCAG 2.x success criteria 1.1.1

<details><summary><i>View the transcript</i></summary>

### 2. Relaxed requirement of alt-text for description in a11y 1.0 

_See github issue [#1441](https://github.com/w3c/epub-specs/issues/1441)._




**Avneesh Singh:** if an image is complex, authors should be using long desc or extended desc.  
… we relaxed this requirement originally because we didn't want to overburden authors.  
… but over the past 4 years, a11y has come a long way.  

**Charles LaPierre:** for Benetech GCA, we're seeing all publishers and vendors embracing extended desc for complex images.  
… we're not getting push back.  

> *Bill Kasdorf:* That is really good to hear!!.

> **Proposed resolution: align requirement of text equivalent in EPUB accessibility with WCAG 2.x SC 1.1.1.** *(Avneesh Singh)*

> *Matthew Chan:* +1.

> *Laura Brady:* +1.

> *Bill Kasdorf:* +1.

> *Ben Schroeter:* +1.

> *Charles LaPierre:* +1.

> *Jun’Ichi Yoshii:* +1.

> *George Kerscher:* +1.

> *Juliette McShane:* +1.

</details>

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2020-12-10-epub-a11y#section2) on 2020-12-10)

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

### 2. Relaxed requirement of alt-text for description in a11y 1.0 

_See github issue [#1441](https://github.com/w3c/epub-specs/issues/1441)._




**Avneesh Singh:** if an image is complex, authors should be using long desc or extended desc.  
… we relaxed this requirement originally because we didn't want to overburden authors.  
… but over the past 4 years, a11y has come a long way.  

**Charles LaPierre:** for Benetech GCA, we're seeing all publishers and vendors embracing extended desc for complex images.  
… we're not getting push back.  

> *Bill Kasdorf:* That is really good to hear!!.

> **Proposed resolution: align requirement of text equivalent in EPUB accessibility with WCAG 2.x SC 1.1.1.** *(Avneesh Singh)*

> *Matthew Chan:* +1.

> *Laura Brady:* +1.

> *Bill Kasdorf:* +1.

> *Ben Schroeter:* +1.

> *Charles LaPierre:* +1.

> *Jun’Ichi Yoshii:* +1.

> *George Kerscher:* +1.

> *Juliette McShane:* +1.

</details>
